code/game/dna/genes/powers.dm

```diff
@@ -236,7 +236,12 @@
 
 	activate(var/mob/M, var/connected, var/flags)
 		..(M,connected,flags)
-		M.pass_flags |= 1
+		M.pass_flags |= PASSTABLE
+
+	deactivate(var/mob/M, var/connected, var/flags)
+		..(M,connected,flags)
```

Comment:
no point with putting args into ```..()``` unless you're changing them from the arguments of the current proc.
Since ```..()``` automatically uses the arguments of the current proc.